src/collect/getRawCoverage.ts

```diff
@@ -4,12 +4,21 @@ import { exec } from '@actions/exec';
 import { readFile, rmdir } from 'fs-extra';
 
 import { REPORT_PATH } from '../constants/REPORT_PATH';
-import { SkipStepType } from '../typings/Options';
+import { PackageManagerType, SkipStepType } from '../typings/Options';
 import { FailReason } from '../typings/Report';
 
 const joinPaths = (...segments: Array<string | undefined>) =>
     join(...(segments as string[]).filter((segment) => segment !== undefined));
 
+const getPackageManagerInstallCommand = (
+    packageManager: PackageManagerType
+): string =>
+    packageManager === 'npm'
+        ? 'npm i'
+        : packageManager === 'yarn'
+        ? 'yarn install'
+        : '';
```

Comment:
`npm i` is actually shorthand for the `npm install` command, so this function can be:

```ts
const getPackageManagerInstallCommand = (
    packageManager: PackageManagerType
): string => `${packageManager} install`;
```

Reply:
Done!